lib/ws-server.js

```diff
@@ -0,0 +1,68 @@
+module.exports = WebSocketServer;
+
+var assert = require('assert');
+var debug = require('./debug')('process:' + process.pid);
+var message = require('./message');
+var util = require('util');
+var url = require('url');
+var Server = require('ws').Server;
+var WebSocketChannel = require('./ws-channel');
+
+
+function WebSocketServer(options, callback) {
```

Comment:
Just like the `http/net.createServer`, the websocket server constructor takes an optional callback which is attached to the 'connection' event. I just pass it along to the base class constructor.